drivers/mtd/spi/winbond-otp.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2016 Imagination Technologies
+ * Author: Avinash Tahakik <avinash.tahakik@imgtec.com>
+ *
+ * SPDX-License-Identifier:	GPL-2.0+
+ */
+
+#include <common.h>
+#include <spi.h>
+#include <spi_flash.h>
+#include <winbond-otp.h>
+
+#include "sf_internal.h"
+
+/* SPI FLASH opcodes */
+
+#define SPINOR_OP_RD_SECURITY_REG	0x48 /* Read security register */
+#define SECURITY_REG_SIZE			256 /* bytes per security register */
```

Comment:
This is because vi and atom are interpreting tabs as 4 spaces. github is using 8 spaces per tab.